SPECS/libtool/libtool.spec

```
@@ -79,16 +87,31 @@ make %{?_smp_mflags} check
 %{_libdir}/libltdl.so.7.3.1
 
 %changelog
+* Mon Aug 30 2021 Bala <balakumaran.kannan@microsoft.com> - 2.4.6-8
+- Replaced build commands with rpm macros
+- Corrected version in previous changelog
+- Removed sha1 macro
```

```suggestion
- License verified
- Removed sha1 macro
```

I already verified it, you can safely take this change. It's not required for this PR btw., just something I noticed now.